packages/core/src/codemods/emotion-to-compiled/__tests__/emotion-to-compiled.test.tsx

```diff
@@ -0,0 +1,276 @@
+jest.disableAutomock();
+
+const defineInlineTest = require('jscodeshift/dist/testUtils').defineInlineTest;
+
+import transformer from '../emotion-to-compiled';
+
+describe('emotion-to-compiled transformer', () => {
+  defineInlineTest(
+    transformer,
+    {},
+    "import styled from '@emotion/styled';",
+    "import { styled } from '@compiled/core';",
+    'it transforms default @emotion/styled imports'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    "import sc from '@emotion/styled';",
+    "import { styled as sc } from '@compiled/core';",
+    'it transforms default with different name than "styled" @emotion/styled imports'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    `
+    /** @jsx jsx */
+    import { css, jsx } from '@emotion/core';
+    `,
+    "import '@compiled/core';",
+    'it transforms all named @emotion/core imports'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    `
+    /** @jsx jsx */
+    import { css as c, jsx } from '@emotion/core';
+    `,
+    "import '@compiled/core';",
+    'it transforms all named @emotion/core imports with different imported name'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    `
+    /** @jsx jsx */
+    import { css, jsx } from '@emotion/core';
+    import styled from '@emotion/styled';
+    `,
+    "import { styled } from '@compiled/core';",
+    'it transforms all named @emotion/core and default @emotion/styled imports'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    `
+    /** @jsx jsx */
+    import { css as c, jsx } from '@emotion/core';
+    import sc from '@emotion/styled';
+    `,
+    "import { styled as sc } from '@compiled/core';",
+    'it transforms all named @emotion/core with different imported name and default with different name than "styled" @emotion/styled imports'
+  );
+
+  defineInlineTest(
+    transformer,
+    {},
+    `
+    /** @jsx jsx */
+    import { css, jsx } from '@emotion/core';
+    import styled from '@emotion/styled';
+    import react from 'react';
+
+    const Component = (props) => (
+      <>
+        <div
+          css={css\`
+            color: red;
+            background-color: #000;
+          \`}
+        />
+        <span css={css\` color: blue; \`} />
+      </>
+    );
+    `,
+    `
+    import { styled } from '@compiled/core';
+    import react from 'react';
+
+    const Component = (props) => (
+      <>
+        <div
+          css={\`
+            color: red;
+            background-color: #000;
+          \`}
+        />
+        <span css={\` color: blue; \`} />
```

Comment:
we have a issue here #313 to add `css` to improve syntax highlighting (and also enable mixins) - we will need to come bask and update the codemod when we implement it - or alternatively - should we not codemod it for now? 🤔

Reply:
😅 Lets do it for this release. I ll leave a comment on #313 or probably create a card? What you think? Its just matter of removing the code.

Reply:
sounds good

we might need to add a codemod to add it back too hehe

Reply:
🤔 hehe yeah. Atleast we will have first release 🎉

Reply:
https://github.com/atlassian-labs/compiled/issues/340